alacritty_terminal/src/tty/windows/conpty.rs

```diff
@@ -20,9 +23,63 @@ use crate::event::{OnResize, WindowSize};
 use crate::tty::windows::child::ChildExitWatcher;
 use crate::tty::windows::{cmdline, win32_string, Pty};
 
+/// Load the pseudoconsole API from conpty.dll if possible, otherwise use the
+/// standard Windows API. The conpty.dll from the Windows Terminal project
+/// supports loading OpenConsole.exe, which offers many improvements and
+/// bugfixes compared to the standard conpty that ships with Windows.
+///
+/// In particular, the new conpty.dll and OpenConsole.exe enable mouse
+/// reporting, which not yet supported by the Windows API on Windows 10 as of
+/// June 2023.
+///
+/// The conpty.dll and OpenConsole.exe files will be searched in PATH and in
+/// the directory where Alacritty executable is located.
```

Comment:
```suggestion
/// Load the pseudoconsole API from conpty.dll if possible, otherwise use the
/// standard Windows API.
///
/// The conpty.dll from the Windows Terminal project
/// supports loading OpenConsole.exe, which offers many improvements and
/// bugfixes compared to the standard conpty that ships with Windows.
///
/// The conpty.dll and OpenConsole.exe files will be searched in PATH and in
/// the directory where Alacritty's executable is located.
```